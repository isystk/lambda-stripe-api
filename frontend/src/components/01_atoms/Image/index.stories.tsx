import { Meta, Story } from '@storybook/react'
import React from 'react'
import Image from './index'

export default {
  title: '01_atoms/Image',
  component: Image,
} as Meta

const Template: Story = (props) => {
  return (
    <div className="w-64 rounded-lg overflow-hidden">
      <Image
        src="/images/dummy_480x320.png"
        className="zoom"
        alt="dummy"
        zoom={true}
      />
    </div>
  )
}

export const Primary = Template.bind({})
Primary.storyName = 'プライマリ'
Primary.args = {}
