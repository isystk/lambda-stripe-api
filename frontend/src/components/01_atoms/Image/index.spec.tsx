import React from 'react'
import renderer from 'react-test-renderer'
import Image from './index'

describe('Image', () => {
  it('Match Snapshot', () => {
    const component = renderer.create(
      <div className="w-64 rounded-lg overflow-hidden">
        <Image src="/images/dummy_480x320.png" className="zoom" alt="dummy" />
      </div>
    )
    const tree = component.toJSON()

    expect(tree).toMatchSnapshot()
    expect(true)
  })
})
