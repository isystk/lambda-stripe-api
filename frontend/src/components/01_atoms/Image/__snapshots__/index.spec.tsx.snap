// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image Match Snapshot 1`] = `
<div
  className="w-64 rounded-lg overflow-hidden"
>
  <div
    className="zoom"
  >
    <img
      alt="dummy"
      className="css-s08p0c "
      loading="lazy"
      src="/images/dummy_480x320.png"
    />
  </div>
</div>
`;
